styles.css

```diff
@@ -604,3 +604,98 @@ img {
   vertical-align: middle;
   display: inline-block;
 }
+
+
+/* Affiliates Page */
+
+.motto-wrap {
+  width: 80%;
+  margin-right: auto;
+  margin-bottom: 0px;
+  margin-left: auto;
+  padding-top: 49px;
+  text-align: center;
+}
+
+.label-light {
+  margin-bottom: 10px;
+  font-size: 12px;
+  line-height: 20px;
+  font-weight: 500;
+  letter-spacing: 1px;
+  text-transform: uppercase;
+  opacity: 0.6;
+}
+
+.affiliates-heading {
+  margin-top: 10px;
+  margin-bottom: 15px;
+  font-size: 36px;
+  line-height: 50px;
+  font-weight: 400;
+  text-transform: none;
+}
+
+.affiliates-content-wrap {
+    margin-top: 80px;
+    margin-bottom: 120px;
+}
+
+.affiliates-layout-grid {
+  display: -ms-grid;
+  display: grid;
+  grid-auto-columns: 1fr;
+  -ms-grid-columns: 1fr 1fr;
+  grid-template-columns: 1fr 1fr;
+  -ms-grid-rows: auto auto;
+  grid-template-rows: auto auto;
+  grid-row-gap: 16px;
+  grid-column-gap: 16px;
+}
+
+.affiliates-about-grid {
```

Comment:
why mention `grid` in the selector name if its not a `grid`. It might be grid children but is it a grid?